src/server/routers/order.route.ts

```diff
@@ -0,0 +1,84 @@
+import express from "express";
+import { OrderItemModel } from "../schemas/order-items.schema.js";
+import { OrderModel } from "../schemas/order.schema.js";
+export const orderRouter = express.Router();
+
+orderRouter.get('/', async(req, res) => {
+    const orderList = await OrderModel.find()
+
+    if(!orderList) {
```

Comment:
i think oderList is always truthy because an empty array evaluates to true.. so I don't know how much function this has .. and if it did return false .. wouldnt it be responding twice? so you might get that "cannot set headers after sent" error or what ever ??

Reply:
Yeah you're right I completely forgot about truthy values but I updated the code to just test the length of orderLIst and yeah should be fine now. thanks